editor/src/clj/editor/lsp/server.clj

```diff
@@ -149,6 +152,9 @@
                          :else
                          (throw (ex-info "Invalid text document sync kind" {:value textDocumentSync})))})
 
+(defn- configuration-handler [{:keys [items]}]
+  (vec (repeat (count items) nil)))
```

Comment:
Dummy configuration helper for lua-language-server that asks for configuration even though we don't define such capability.